src/screens/WebScreen.js

```diff
@@ -17,13 +18,21 @@ export const WebScreen = ({ navigation, route }) => {
   const trackScreenViewAsync = useTrackScreenViewAsync();
   const webUrl = route.params?.webUrl ?? '';
   const injectedJavaScript = route.params?.injectedJavaScript ?? '';
+  const isExternal = route.params?.isExternal ?? '';
 
   // NOTE: we cannot use the `useMatomoTrackScreenView` hook here, as we need the `webUrl`
   //       dependency
   useEffect(() => {
     isConnected && webUrl && trackScreenViewAsync(`${MATOMO_TRACKING.SCREEN_VIEW.WEB} / ${webUrl}`);
   }, [webUrl]);
 
+  useEffect(() => {
+    if (isExternal) {
+      openLink(webUrl);
+      navigation.goBack();
+    }
+  }, []);
+
```

Comment:
i think we should add `isExternal` here `if (!webUrl) return null;` to ensure, that nothing is tried to be rendered for these cases. i do not know how react behaves before navigating. with rendering `null` we know, that nothing happens that we do not want, right?!

Reply:
Yes, that's right. I will update the code as follows. 
Because as you said, the page will be rendered until goBack() runs, maybe even for a short time. I think this will work with an invisible speed, but it will still be an unnecessary process.

```
  if (!webUrl && isExternal) return null;
```

Reply:
no, it should be an or with ||. you will probably never have no web url and is external at the same time

Reply:
Yes, sorry. You're right 👍